subjects/devops/in-back-ground/README.md

```diff
@@ -0,0 +1,55 @@
+## in-back-ground
+
+### Instructions
+
+Create a script `in-back-ground.sh` which will execute a job in the background that will do the following:
+
+- Run the command `cat` on the file "facts" which will read the contents of the file and print it to `stdout`.
+- The output of the cat command will be piped to the `grep` command, which will search for the string `"moon"` in the file.
+- If the `grep` command succeeds (if it finds the string "moon"), the echo command will run and print `"The moon fact was found!"` to the `output` file.
+- If the command finds the string "moon", it will print the matching line to the `stdout`.
+- Redirect the `stdout` to a new file `output`.
+
+You must do all these steps running only one job and using the command `nohup`!
+
+Expected output:
+
+```console
```

Comment:
the output of the current solution is not coherent with the subject - the following is the current solution output on my machine
```console
$  bash solutions/in-back-ground.sh 
nohup: redirecting stderr to stdout
```

Reply:
the file"facts" is now only created in the test so if you try to run the program from the solution folder it will give you that output.
To test the solution you need to create a 'facts' file... actually, i will add a tip in the subject  about this...